src/components/ShoppingCart/dropDownButton.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+import { GrDown, GrUp } from "react-icons/gr";
+import { IconButton } from "@chakra-ui/react";
+import { Collapse } from "@chakra-ui/react";
+import { PriceBar } from "@/store/reducer/priceBarSlice";
+
+type detail = {
+  detail: PriceBar[];
+};
+
+const DropDownButton = ({ detail }: detail) => {
+  const [show, setShow] = useState(false);
+
+  const handleToggle = () => setShow(!show);
+
+  const scroll = "scroll";
+  const hidden = "hidden";
+  const noWrap = "nowrap";
+  const normal = "normal";
+  const clip = "clip";
+  const ellipsis = "ellipsis";
+
+  return (
+    <div
```

Comment:
I think you can use some chakra components (such as Box) instead of the div element.

Reply:
Thank you. I have fixed that.